dbt-bigquery/tests/unit/test_bigquery_connection_manager.py

```diff
@@ -128,12 +128,24 @@ def test_copy_bq_table_truncates(self):
             kwargs["job_config"].write_disposition, dbt.adapters.bigquery.impl.WRITE_TRUNCATE
         )
 
-    def test_job_labels_valid_json(self):
+    @patch("dbt.adapters.bigquery.connections.get_invocation_id", new=Mock(return_value="uuid4"))
+    @patch.object(BigQueryConnectionManager, "get_labels_from_query_comment")
+    def test_get_job_labels(self, mock_get_query_comment_labels):
+        query_labels = {"key": "value"}
+        expected = {**query_labels, "dbt_invocation_id": "uuid4"}
+        mock_get_query_comment_labels.return_value = self.connections._labels_from_query_comment(
```

Comment:
mocking get_labels_from_query_comment to keep the test simple and unit test only get_job_labels, in another case have to create many obj for fully operational BigQueryConnectionManager instance. (https://github.com/dbt-labs/dbt-adapters/blob/main/dbt-bigquery/src/dbt/adapters/bigquery/connections.py#L213)